### ISSUE TITLE
!

### ISSUE THREAD

**Issue by mediainfocl** [jailsonsb2/RadioPlayer#10]
THIS IS A COPIED PLAYER!, THE ORIGINAL IS FROM MY WEBSITE AND MY AUTHORSHIP: https://insidefm.cl . Have the decency to be original and at least change the style of the player, but just don't copy it!.

ORIGINAL PLAYER: https://insidefm.cl

CREATED BY: MEDIAINFO COMUNICACIONES & ZKREATIONS!.

**Comment by jailsonsb2** [comment]
This code is made available by the original authors, friend, it is credited in the readme, I just fixed what they stopped working. Give me credit in the API code that you are using and stop talking this nonsense.

**Comment by jailsonsb2** [comment]
![image](https://github.com/user-attachments/assets/6304c7c9-c9bf-48f5-b8e2-da6bc8b160d0)

**Comment by mediainfocl** [comment]
Excuse me? Your API? I didn't know that your API shows all the information that my API shows. I also didn't know that your API works with Deezer, iTunes and Spotify. From what I see, your API doesn't look anything like mine.

I didn't know that I had made my player code available to the public.

IT IS A UNIQUE PLAYER AND DEVELOPED FOR MEDIAINFO COMUNICACIONES AND INSIDEFM (PLAYER DEVELOPER: ZKREATIONS)

**Comment by jailsonsb2** [comment]
Just simulate an error. It will show the same error as mine. You should have done your homework. A copy with something extra is still a copy, you should be grateful because your players only work thanks to me. Whether you recognize it or not.

**Comment by jailsonsb2** [comment]
![image](https://github.com/user-attachments/assets/1d09f1a7-8554-46a5-bf25-98272be3a69d)

**Comment by mediainfocl** [comment]
At least I had the decency to modify the code and add more things and improve it. You just copied and pasted. As I said (since all your players are copied) at least modify the style of the player and make it unique! I'm not angry that you stole my player, I'm upset that you didn't even change the style of the player. As I said, at least I have the decency to write new code and improve yours, which was just very basic.

**Comment by jailsonsb2** [comment]
I really don't care, you can come here and spout whatever nonsense you want. You and I both know the truth, you can continue your act. Take your 5 minutes of fame and be happy. I'm glad you were able to use my code and customize it and now you can make money selling your players.

**Comment by mediainfocl** [comment]
THAT'S THE PROBLEM! I DON'T SELL MY PLAYERS. I CREATE THEM UNIQUELY AND EXCLUSIVELY FOR MY WEBSITES. I'm not interested in profiting from your "API", that's why I have it for free use. I'm not interested in selling my players. That's why it bothers me that it's a vile copy of my player. HAVE SOME DECENCY IN YOUR LIFE, MAN!

**Comment by jailsonsb2** [comment]
First you say that you developed the API, then you admitted that you copied my code and added functions, I think it's already proven for everyone to see who really is who. Be happy, friend. You won't have my attention anymore, I've already proven what I wanted.

**Comment by mediainfocl** [comment]
I never said I developed the API, I just said I developed the improvements and there is a big difference. Also, are you sure you developed the API? Curiously, Stream Africa (joey) offered the API long before yours. It is also curious that I implemented improvements to my API and you appeared with those improvements... oops, but you really missed some improvements, a simple basic API that works with iTunes, wouldn't you like your API to work with Deezer, Spotify and Tidal too?
